src/Http/src/Stream/GeneratorStream.php

```diff
@@ -20,16 +18,15 @@ final class GeneratorStream implements StreamInterface
 
     private bool $started = false;
 
-    public function __construct(Generator $body)
+    public function __construct(private ?Generator $stream)
```

Comment:
We can't rename parameters in public signatures in v3

Reply:
sure, reverted, updated the configured rule as disallow rename property https://github.com/spiral/framework/pull/1139/commits/ccd058aae09a77189f31f1b45b20203664efb925

Reply:
sure, reverted, updated the configured rule as disallow rename property https://github.com/spiral/framework/pull/1139/commits/ccd058aae09a77189f31f1b45b20203664efb925